config/crd/bases/components.platform.opendatahub.io_datasciencepipelines.yaml

```diff
@@ -49,6 +49,26 @@ spec:
           spec:
             description: DataSciencePipelinesSpec defines the desired state of DataSciencePipelines
             properties:
+              argoWorkflowsControllers:
+                properties:
+                  managementState:
+                    default: Managed
+                    description: |-
+                      Set to one of the following values:
+
+                      - "Managed" : the operator is actively managing the bundled Argo Workflows controllers.
+                                    It will only upgrade the Argo Workflows controllers if it is safe to do so. This is the default
+                                    behavior.
+
+                      - "Removed" : the operator is not managing the bundled Argo Workflows controllers and will not install it.
+                                    If it is installed, the operator will remove it but will not remove other Argo Workflows
+                                    installations.
+                    enum:
+                    - Managed
+                    - Removed
+                    pattern: ^(Managed|Unmanaged|Force|Removed)$
```

Comment:
given the valid values of just "Managed" and "Removed", should `pattern` be updated to only include those?